rustworkx-core/src/centrality.rs

```diff
@@ -1095,42 +1095,50 @@ mod test_katz_centrality {
 /// let dg = petgraph::graph::DiGraph::<i32, ()>::from_edges(&[
 ///     (0, 4), (1, 2), (2, 3), (3, 4), (1, 4)
 /// ]);
-/// let output = closeness_centrality(&dg, true);
+/// let output = closeness_centrality(&dg, true, 200);
 /// assert_eq!(
 ///     vec![Some(0.), Some(0.), Some(1./4.), Some(1./3.), Some(4./5.)],
 ///     output
 /// );
 /// ```
-pub fn closeness_centrality<G>(graph: G, wf_improved: bool) -> Vec<Option<f64>>
```

Comment:
Do you want to mention in the docstring that this function runs in parallel now?

Reply:
Done